.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,3 +19,6 @@ along with its entire output.
 Even for runtime issues it helps a lot if you can include the full link command.
 Adding `-v` to the link command will show all of the sub-commands run which
 can help us diagnose your issue.
+
+Note:  Where possible, please avoid attaching screen shots of code or console
+logs.  Instead, please include them text so that they may be copied / searched.
```

Comment:
```suggestion
logs.  Instead, please include them as text so that they may be copied / searched.
If you want them to be syntax highlighted, escape them with three backticks before and after,  like this:
```cpp
int x = 20;
```
(I can't figure out how to add the closing backticks due to escaping, but adding them in the repo should work.)